### PR TITLE
Add qrcode icon

### PR DESCRIPTION
**Pull request by tombertrand**
## Description

Add `qrcode` icon

## How to test?

- Checkout branch, run `yarn dev`
- Open [Storybook](http://localhost:6006)
- Or check the deploy preview on Netlify (link available in comments)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/lightspeed/flame/blob/master/.github/CONTRIBUTING.md) guide
- [ ] I have prepared [CHANGELOGs](https://github.com/lightspeed/flame/blob/master/.github/CONTRIBUTING.md#git-workflow) for release
- [ ] I have tested my changes on [supported browsers](https://browserl.ist/?q=%3E0.25%25%2C+not+op_mini+all%2C+not+ie+%3C%3D+11)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/flame/svg/Icons/qrcode.svg [hunk 1]
```diff
@@ -0,0 +1,6 @@
...
+    </g>
+</svg>
```
Comment (glambert) [inline packages/flame/svg/Icons/qrcode.svg]: Raname file to `icon-qrcode.svg` to match other icon names.

File: packages/flame/svg/Icons/qrcode.svg [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
....4298,0.4298l0,3.46912c0,0.24559 0.1842,0.4298 0.4298,0.4298l3.46912,0c0.24559,0 0.4298,-0.1842 0.4298,-0.4298zm1.72943,-3.03932l2.59928,0l0,2.59928l-2.59928,0l0,-2.59928zm1.84201,1.84201l0,-1.08473l-1.08473,0l0,1.08473l1.08473,0zm-5.19855,0l0,-1.08473l-1.08473,0l0,1.08473l1.08473,0zm-1.84201,3.35654l2.59928,0l0,2.59928l-2.59928,0l0,-2.59928zm8.53463,0.34793l0,-1.08473l-1.0745,0l0,1.08473l1.0745,0zm-2.98815,0l0,-1.08473l-1.09496,0l0,1.08473l1.09496,0zm-3.70448,1.49407l0,-1.08473l-1.08473,0l0,1.08473l1.08473,0zm5.19855,0l0,-1.08473l-1.08473,0l0,1.08473l1.08473,0zm-1.49407,1.49407l0,-1.07451l-1.09496,0l0,1.07451l1.09496,0zm2.98815,0l0,-1.07451l-1.0745,0l0,1.07451l1.0745,0zm0.13303,-5.49531l0,-3.46912c0,-0.24559 -0.19444,-0.4298 -0.44004,-0.4298l-3.45888,0c-0.25583,0 -0.44004,0.1842 -0.44004,0.4298l0,3.46912c0,0.24559 0.1842,0.4298 0.44004,0.4298l3.45888,0c0.24559,0 0.44004,-0.1842 0.44004,-0.4298zm-8.66767,-3.03932l2.59928,0l0,2.59928l-2.59928,0l0,-2.59928zm3.46912,8.22763l0,-3.45888c0,-0.25583 -0.1842,-0.44004 -0.4298,-0.44004l-3.46912,0c-0.24559,0 -0.4298,0.1842 -0.4298,0.44004l0,3.45888c0,0.24559 0.1842,0.44004 0.4298,0.44004l3.46912,0c0.24559,0 0.4298,-0.19444 0.4298,-0.44004z"/>
```
Comment (glambert) [inline packages/flame/svg/Icons/qrcode.svg]: Not sure if it’s GitHub having issues with syntax highlighting or if the svg is not valid, might be worth double-checking.

**Review by glambert — APPROVED** [review]
Two small things the LGTM :tada:

File: packages/flame/svg/Icons/qrcode.svg (hunk 2 above)
Comment (tombertrand) [reply packages/flame/svg/Icons/qrcode.svg]: not sure either (syntax is good)... looking good in dev
image

File: packages/flame/svg/Icons/qrcode.svg (hunk 1 above)
Comment (tombertrand) [reply packages/flame/svg/Icons/qrcode.svg]: good catch 👍

**Pull request merged**